test_cachediff.py

```diff
@@ -35,7 +35,8 @@ def test_has_virtual_address(self):
 
 class TestFile(unittest.TestCase):
     def setUp(self):
-        file_path = os.path.join(os.getcwd(), 'test_file.c')
+        #file_path = os.path.join(os.getcwd(), 'test_file.c') since os.getcwd get /home/sameer etc
+        file_path = '/home/saimadhav/cachediff/test_file.c'
```

Comment:
what if the test was run from a different directory? that was why i used os.getcwd + test_file.c